src/App.js

```diff
@@ -0,0 +1,21 @@
+// // src/App.js
+// import React from 'react';
+// import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
+// import Greeting from './components/greeting';
+
+// function App() {
+//   return (
+//     <div>
+//       <Router>
+//         <nav>
+//               <Link to="/greeting">Greeting</Link>
+//         </nav>
+//         <Routes>
+//           <Route path="/greeting" element={<Greeting />} />
+//         </Routes>
+//       </Router>
+//     </div>
+//   );
+// }
+
+// export default App;
```

Comment:
- [ ] You did an excellent job and your code works perfectly 👏 . However, kindly make sure you implement these sections of the code base as it's a major requirement to have an `app` component using the `React-router-dom` dependency. This is also a good practice that should be held.

### Project requirements:
![image](https://github.com/fatima-najafi/hello-react-front-end/assets/50990925/33381e00-f477-41cf-9504-78a7d8f8aa7a)